modelcluster/models.py

```diff
@@ -186,7 +190,10 @@ def save(self, **kwargs):
         super(ClusterableModel, self).save(update_fields=real_update_fields, **kwargs)
 
         for relation in relations_to_commit:
-            getattr(self, relation).commit()
+            try:
+                getattr(self, relation).commit()
+            except AttributeError:
+                pass
```

Comment:
If you're going to catch and discard exceptions, please add a comment and relevant test coverage to show why this is a valid thing to do, and isn't simply hiding a genuine error.

In this case, I don't think it is valid: it's just masking the bug described in https://github.com/torchbox/django-modelcluster/pull/36#issuecomment-237304134 (the lack of `commit` method is an indication that `get_all_child_relations` is incorrectly returning things that aren't `Deferring*RelatedManager`s).